janusgraph-hbase-parent/janusgraph-hbase-098/pom.xml

```diff
@@ -13,7 +13,7 @@
     <properties>
         <jruby.version>1.6.8</jruby.version>
         <zookeeper.version>3.4.6</zookeeper.version>
-        <top.level.basedir>${project.parent.parent.basedir}</top.level.basedir>
+        <top.level.basedir>${basedir}/../..</top.level.basedir>
```

Comment:
Why is this change needed? This replacement pattern is possible in other `pom.xml` files that weren't updated with this PR. It seems like something that would be a consistent change across the repo: `janusgraph-hadoop-parent/pom.xml`, `janusgraph-hbase-parent/janusgraph-hbase/pom.xml`, etc.

Reply:
@pluradj In Travis `${project.parent.parent.basedir}` didn't resolve properly but `${basedir}/../../` did. Originally I only updated in those modules with unit tests since jacoco reports were writing to the wrong spot but I just pushed updates to to use `${basedir}/../..` everywhere consistently.

Reply:
This should work

Reply:
@pluradj Are you ok with this update/change?

Reply:
LGTM +1